src/pages/get-started.tsx

```diff
@@ -66,13 +66,87 @@ const SearchPullListSection = () => {
   );
 };
 
+const RunListContainersSection = () => {
+  return (
+    <section className="bg-blue-500 dark:bg-blue-900 text-white dark:text-white py-6" >
+      <SectionHeader textColor="text-purple-700 dark:text-purple-500" title="Running a container &amp; listing running containers" />
+      <div className="text-center flex flex-col mx-4">
+        <p className="mb-4">This sample container will run a very basic httpd server that serves only its index page.</p>
+        
+        <h3 className="text-purple-700 dark:text-purple-300 mb-2">Running a container</h3>
+
+        <div className="mx-auto">
+          <CodeBlock language="bash" showLineNumbers className="text-left overflow-scroll max-w-full">
+            $ podman run -dt -p 8080:80/tcp docker.io/library/httpd {'\n'}
+          </CodeBlock>
+        </div>
+
+        <div className="flex flex-col sm:flex-row justify-center gap-6 mt-2">
+          <div className="max-w-lg rounded-md bg-white px-6 py-6 shadow-lg dark:bg-gray-900/50">
+            <h5 className="text-gray-700">Note:</h5>
+            <p className="text-gray-700 text-left">
+              Because the container is being run in detached mode, represented by the -d in the podman run command, 
+              Podman will print the container ID after it has executed the command. The -t also adds a pseudo-tty 
+              to run arbitrary commands in an interactive shell.
+            </p>
+          </div>
+          <div className="max-w-lg rounded-md bg-white px-6 py-6 shadow-lg dark:bg-gray-900/50">
+            <h5 className="text-gray-700">Note:</h5>
+            <p className="text-gray-700 text-left">
+            We use port forwarding to be able to access the HTTP server. For successful running at least <strong>slirp4netns</strong> v0.3.0 is needed.
```

Comment:
I think mentioning slirp version is more of an advanced user option, I believe podman usually packages the correct version of slirp by default, this might be confusing in a getting started page

Reply:
We can remove this, 0.3.0 is super old, even RHEL/Centos 7 have 0.4.3 in the repositories.